api/src/opentrons/protocol_api/labware.py

```diff
@@ -341,16 +342,18 @@ def uri(self) -> str:
 
     @property  # type: ignore[misc]
     @requires_version(2, 0)
-    def parent(self) -> Union[str, ModuleTypes, None]:
+    def parent(self) -> Union[str, ModuleTypes, OffDeckType]:
         """The parent of this labware.
 
         If the labware is on the deck, a `str` deck slot name will be returned.
         If on a module, the parent :py:class:`ModuleContext` will be returned.
-        If off deck, `None` will be returned.
+        If off deck, `off-deck` will be returned.
```

Comment:
To reference the actual value that users should use:

```suggestion
        If off deck, :py:obj:`OFF_DECK` will be returned.
```